postgresql/README.md

```diff
@@ -62,48 +132,27 @@ EOF
 docker-compose up
 ```
 
-## Binding
+## Update Strategies
 
-Consuming services can bind to PostgreSQL via:
+For more information on update strategies, see [this documentation](https://www.habitat.sh/docs/using-habitat/#update-strategy)
 
-```
-hab start <origin>/<app> --bind database:postgresql.default --peer <pg-host>
-```
+If you are running a standalone PostgreSQL instance and want to use an update strategy, you should use either "none" (which is the default update strategy) or "all-at-once".
 
-Superuser access is exposed to consumers when binding and we advise that required databases, schemas and roles be created and migrations get run in the init hook of the consuming service. The created roles (with restricted access) should then be exposed to the application code that will get run.
+If you are running a PostgreSQL cluster, you will likely want to use the Rolling strategy.
 
-In the template of the consuming service, there is currently a difference between the binding syntax for binding to the leader in a clustered topology vs binding to any member.  A [future enhancement](https://github.com/habitat-sh/habitat/issues/4127) may improve upon that, but for now something like this could work if used in an interpreted script file:
+## Scaling
 
-```
-# Use the eachAlive and @first helpers to bind to the oldest alive member of the bound service group
-#   - members are processed in the chronological order of when they joined the ring
-# If a later member comes along that is the leader of a cluster topology, override the environment variable definitions with that
-#   - if not, the second section is omitted
-{{~ #if bind.database}}
-  {{~ #eachAlive bind.database.members as |member|}}
-    {{~ #if @first}}
-# First alive member
-PGHOST="{{member.sys.ip}}"
-PGPORT="{{member.cfg.port}}"
-PGUSER="{{member.cfg.superuser_name}}"
-PGPASSWORD="{{member.cfg.superuser_password}}"
-    {{~ /if}}
+This plan can be run in a cluster, see the above section on "Leader/Follower" for more information.
 
-    {{~ #if member.leader}}
-# Leader node override
-PGHOST="{{member.sys.ip}}"
-PGPORT="{{member.cfg.port}}"
-PGUSER="{{member.cfg.superuser_name}}"
-PGPASSWORD="{{member.cfg.superuser_password}}"
-    {{~ /if}}
-  {{~ /eachAlive}}
-{{~ /if}}
-```
+This plan has not been tested in a highly available architecture.
 
-However if you're using more of a rigid configuration file syntax then you still need to know and choose in advance whether or not you're expecting to connect to a leader topology or not.
+## Monitoring
+
+If you wish to monitor this service directly, you can use [Datadog](https://www.datadoghq.com/) (there is also a [datadog agent plan](./dd-agent) in this repo), [Sumologic](https://www.sumologic.com/)(there is also a [sumlogic plan](./sumologic)), and other monitoring services.
 
+## Notes
 
-# TODO (Potential improvements to this plan):
+### TODO (Potential improvements to this plan):
```

Comment:
Test for various high-availability and other operational optimizations (high writes, high reads), and add configuration to the plan for people who wish to use those.